docs-next/pages/apis/hooks.mdx

```diff
@@ -2,6 +2,455 @@ import { Markdown } from '../../components/Page';
 
 # Hooks API
 
-(coming soon)
+Hooks allow you to execute code at different stages of the mutation lifecycle when performing create, update, and delete operations.
+Lists and fields both support the same set of hook functions, with some slight differences in the arguments they accept.
+The differences will be explicitly called out below.
+For each hook, the fields hooks are applied to all fields in parallel first, followed by the list hooks.
```

Comment:
This line is super important. Can we format it differently so it gets called out more clearly?

e.g

```suggestion

!> For each hook, the fields hooks are applied to **all fields first** in parallel, followed by the list hooks.
```

tbh a diagram on this page would go a long way, I guess that's more for the guide though, and we can come back to that another time.